space-template.yaml

```diff
@@ -0,0 +1,166 @@
+    type: *list
+    element_type: *translatable_string
+    default_value: *state_new
+    values: *common_work_item_states
+    required: false
+
+# If no other fields are required for your work item you can use this handy default
+common_work_item_fields: &common_work_item_fields
+  system_title: *field_title
+  system_description: *field_description
+  system_created_at: *field_created_at
+  system_updated_at: *field_updated_at
+  system_state: *field_state
+
+# This map defines all the work item types that exist in a space. We re-use some
+# of the above states and lists in order share code between work item types.
+work_item_types:
+  &wit_bug bug:
+    title:
+      en_US: "Bug"
+      de_DE: "Fehler"
+    description:
+      en_US: "A software bug is an error, flaw, failure or fault in a computer program"
+      de_DE: "Ein Fehler bezeichnet im Allgemeinen ein Fehlverhalten von Computerprogrammen"
+    fields: *common_work_item_fields
+  &wit_feature feature:
+    title:
+      en_US: "Feature"
+      de_DE: "Fähigkeit"
+    description:
+      en_US: "TBD"
+      de_DE: "TBD"
+    fields: *common_work_item_fields
+  &wit_epic epic:
+    title:
+      en_US: "Epic"
+      de_DE: "Epik"
+    description:
+      en_US: "TBD"
+      de_DE: "TBD"
+    fields: *common_work_item_fields
+  &wit_user_story user_story:
+    title:
+      en_US: "User story"
+      de_DE: "Anwendererzählung"
+    description:
+      en_US: "A way to describe a scenario such that it has clear acceptance 
+      criteria and that unambiguously describes a user interaction , or the
+      interaction between two parts of a system. Team members tak a user story
+      and decompose it into tasks that satisfy the user story. Usually some
+      point value is assigned to a story that serves a way to measure the
+      rate of completing user stories."
+      de_DE: ""
+    fields: *common_work_item_fields
+  &wit_task task:
+    title:
+      en_US: "Task"
+      de_DE: "Aufgabe"
+    description:
+      en_US: "TBD"
+      de_DE: "TBD"
+    fields: *common_work_item_fields
+  &wit_scenario scenario:
+    title:
+      en_US: "Scenario"
+      de_DE: "Scenario"
+    description:
+      en_US: "A specific defined interaction/sequence of interactions with a
+             system to achieve a given goal"
+      de_DE: "Eine speziell definierte Interaktion/Sequenz von Interaktionen
+             mit einem System, um ein Ziel zu erreichen"
+    fields: *common_work_item_fields
+  &wit_experience experience:
+    title:
+      en_US: "Experience"
+      de_DE: "Erfahrung"
+    description:
+      en_US: "TBD"
+      de_DE: "TBD"
+    fields: *common_work_item_fields
+
+work_item_categories:
+    requirements: [*wit_task, *wit_bug]
+    board: [*wit_user_story, *wit_task, *wit_bug]
+    root_level: [*wit_scenario, *wit_experience]
+    backlog: [ *wit_user_story, *wit_task, *wit_bug]
+
+link_types:
+    hierarchy:
+        - source: *wit_epic
```

Comment:
@maxandersen like @aslakknutsen said. In addition we should allow users to define a parent WIT for their own WITs. For example:

```
planneritem/
├── customer-A-base
│   ├── bug
│   ├── epic
│   └── task
└── customer-B-base
    ├── bug
    ├── epic
    └── task
```

That would allow them to link from any `customer-A-base` sub-type to any other `customer-A-base` sub-type if both ends of the link (source and target) are of type `customer-A-base`.

Do we want to introduce some kind of a `parent` field in the YAML for this? In core I think we copy all the fields from the parent type to the new type in order to make them compatible. That means you probably must skip this line if it is already on the base type:

```yaml
fields: *common_work_item_fields
```